### DISCUSSION TITLE
How can I get Galaxy Brain achievement?

### DISCUSSION THREAD

**Discussion by NFTknight** [community/community#84922] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Who can help me with getting achievement of Galaxy Brain?

**Comment by everythinguwannatobe** [comment]
You can refer this link.
This link explains how to get that step by step.

**Reply by ahlyel-amine** [reply]
thanks

**Comment by rezzcode** [comment]
To get a galaxy brain badge you mast get two of your answers or comments marked as answers,
that can be done by taking part in discussions, providing help and answerers to as many discussions as possible.

for more on how to get the other badges you can check out on `github docs` or at this link [github badges](https://github.com/github-profile-achievements/english) which has a documentation provided in different languages.

Soon  you will be able to have something like this:

![image](https://github.com/github-profile-achievements/english/assets/154701770/5d3a44e7-c4ab-44d9-8a6d-57ad5d731bb8)

**hope it helps.**

**Reply by NFTknight** [reply]
Thanks

**Reply by hotcoding85** [reply]
thanks @the-1Riddle

**Reply by Yadzim** [reply]
Thanks 👍

**Comment by github-staff** [comment]
Hi @NFTknight, thanks so much for posting in our Community. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by rezzcode** [reply]
Mmmmh, that's ok with me.

as it will encourage more focused community discussions and the rate of low quality posts as you say will go down, like once seen a crypto post on the community lol.

👍

**Comment by kyuna0312** [comment]
just answer 2 discussion and accept that answer then get badge i think.
Is it can be a answer xD

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @NFTknight. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by pedrobezerra14** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Who can help me with getting achievement of Galaxy Brain?

just answer 2 discussion and accept that answer then get badge i think.

**Reply by rustor0642** [reply]
thanks

**Comment by itz-predator** [comment]
Thanks

**Comment by rustor0642** [comment]
How can I get Galaxy Brain achievement? 

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by manuel14mds** [comment]
Could someone helping me to achieve it too?

**Comment by Yadzim** [comment]
How to get i this

**Reply by 001123** [reply]
I''ll help you

**Comment by 001123** [comment]
Please help me get this

**Comment by Vaibhav-kesarwani** [comment]
Is some one is helping me to get this badge???

**Reply by NugrahAnggaraS** [reply]
yes

**Comment by ebndev** [comment, marked as answer]
Hello all! Here with an updated answer:

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. If users create discussions of this nature, it may result in a temporary or indefinite block from the Community. 

Thanks for understanding!

**Comment by ahlyel-amine** [comment]
I think that you need to answer other people's discussions to be counted as an answer and not your own discussion.
